packages/embark-graph/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "embark-graph",
+  "version": "4.0.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark is a framework that allows you to easily develop and deploy DApps",
+  "homepage": "https://embark.status.im/",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "license": "MIT",
+  "repository": "github:embark-framework/embark",
+  "main": "./dist/index.js",
+  "scripts": {
```

Comment:
above scripts can add:
```
"files": [
    "dist"
],
```

Reply:
the same can be added to embark-compiler. embark-typings doesn't need it, but embark and embark-ui can have `"files"` too, but theirs will have a little more involved than just `"dist"`.

files should end up working better than npmignore, less prone to "oops".